plugins/labhub.py

```diff
@@ -33,47 +32,54 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-class LabHub(BotPlugin):
+class LabHub(BotPlugin, DefaultConfigMixin):
     """GitHub and GitLab utilities"""  # Ignore QuotesBear
 
-    GH_ORG_NAME = constants.GH_ORG_NAME
-    GL_ORG_NAME = constants.GL_ORG_NAME
+    CONFIG_TEMPLATE = {
+        'GH_ORG_NAME': 'org_name',
```

Comment:
shouldnt these be `None`, and the presense of `None` used to deactivate the plugin ?

useless values are useless.
the plugin will fail.